### PR TITLE
web: add navigate with 1, 2, 3 keys

### PR DESCRIPTION
**Pull request by nicks**
Hello @hyu, @maiamcc,

Please review the following commits I made in branch nicks/morenav:

eb19230b3b3b2b45b1243ea8290a3de1b4792777 (2020-10-06 18:57:45 -0400)
web: add navigate with 1, 2, 3 keys

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

**Comment by nicks** [comment]
still playing around with the react primitives for context and testing

File: web/src/SecondaryNavKeyboardShortcuts.test.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      <SecondaryNavKeyboardShortcuts
+        logUrl={logUrl}
+        alertsUrl={alertsUrl}
+        facetsUrl={facetsUrl}
+      />
+    </MemoryRouter>
+  )
+}
+
+it("navigates to logs", () => {
```
Comment (maiamcc) [inline web/src/SecondaryNavKeyboardShortcuts.test.tsx:27]: again, no strong feelings, but would it be useful to test that eg ctrl+1 doesn't get picked up as a nav event?

File: web/src/SecondaryNavKeyboardShortcuts.test.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  expect(fakeHistory.location.pathname).toEqual("/logs")
+})
+
+it("navigates to alerts", () => {
+  shortcuts("/logs", "/alerts", "/facets")
+  fireEvent.keyDown(document.body, { key: "2" })
+  expect(fakeHistory.location.pathname).toEqual("/alerts")
+})
+
+it("navigates to facets", () => {
```
Comment (maiamcc) [inline web/src/SecondaryNavKeyboardShortcuts.test.tsx:39]: worth a test for "facets url DNE"?

**Review by maiamcc — APPROVED** [review]
...am I missing the place where we actually NAVIGATE to the given URL? 🤔

**Comment by maiamcc** [comment]
...whoops that wasn't supposed to be an approval, cuz I have open Qs.

**Comment by nicks** [comment]
re: navigation, history.push() is how you navigate in-app with the framework we're using - https://github.com/ReactTraining/history/blob/master/docs/navigation.md

File: web/src/SecondaryNavKeyboardShortcuts.test.tsx (hunk 1 above)
Comment (nicks) [reply web/src/SecondaryNavKeyboardShortcuts.test.tsx:27]: added

File: web/src/SecondaryNavKeyboardShortcuts.test.tsx (hunk 2 above)
Comment (nicks) [reply web/src/SecondaryNavKeyboardShortcuts.test.tsx:45]: added

File: web/src/HUD.tsx [hunk 3]
```diff
@@ -430,7 +430,6 @@ class HUD extends Component<HudProps, HudState> {
       return (
         <Sidebar isClosed={isSidebarClosed} toggleSidebar={this.toggleSidebar}>
           <SidebarKeyboardShortcuts
-            history={this.props.history}
```
Comment (maiamcc) [inline web/src/HUD.tsx:433]: why did this go away?

**Review by maiamcc — APPROVED** [review]
ty for the link, this lgtm!

File: web/src/HUD.tsx (hunk 3 above)
Comment (nicks) [reply web/src/HUD.tsx:433]: i switch to using the useHistory() hook, which makes the component a bit more modular, https://github.com/ReactTraining/react-router/blob/15bb45b4d309bc3393659c4f4ded9bb55a03048b/packages/react-router/docs/api/hooks.md#usehistory

**Pull request merged**